pages/api/channelsInfo.ts

```diff
@@ -13,6 +13,16 @@ export default async function handler(
         await APIProvider.api().listNodes(channel.peer_id)
       ).nodes[0];
     }
+    for (const channel of funds.channels) {
+      if (channel.node_info == undefined) {
+        let removeIndex = funds.channels
+          .map(function (item) {
+            return item.peer_id;
+          })
+          .indexOf(channel.peer_id);
+        funds.channels.splice(removeIndex, 1);
+      }
+    }
```

Comment:
You can semplify this wil the following code
```suggestion
    funds = funds.filter((channel, index, array) => channel.channel_info != null); 
```

Reply:
```
funds.channels = funds.channels.filter((channel) => channel.node_info != null); 
```

ig this would work